packages/account/src/configs.ts

```diff
@@ -1,4 +1,10 @@
-export const FUEL_NETWORK_URL =
+// #region FUEL_NETWORK_URL
+export const FUEL_NETWORK_URL: string =
   typeof process !== 'undefined'
     ? process?.env?.FUEL_NETWORK_URL || 'http://127.0.0.1:4000/graphql'
     : 'http://127.0.0.1:4000/graphql';
+// #endregion FUEL_NETWORK_URL
+
+// #region FUEL_BETA_5_NETWORK_URL
+export const FUEL_BETA_5_NETWORK_URL: string = 'https://beta-5.fuel.network/graphql';
+// #endregion FUEL_BETA_5_NETWORK_URL
```

Comment:
It is better to use a constant name that we would not need to change later, like: `FUEL_TESTNET_NETWORK_URL`

Reply:
IMO it's better if it's different, changing the url implies breaking changes, so we should change the constant.

Reply:
I did have `FUEL_TEST_NETWORK_URL` initially, however, I favour the more explicit approach of `FUEL_BETA_5_NETWORK_URL` for the increased clarity to the user.

Reply:
I believe it's unnecessary to maintain a constant for the specific name of the current beta testnet. Things will break